### PR TITLE
time_granularity -> freq

### PR DESCRIPTION
**Pull request by lostella**
*Issue #, if available:* #134 

*Description of changes:* s/time_granularity/freq/g

**NOTE:** this is a breaking change.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lostella** [comment]
> Can you just check that 'time_granularity' is gone from every folder?

I did. However, it looks like the host producing the docs is now failing, because in the notebook there is `"regenerate=False"`, so probably the datasets are kept on the machine and the metadata file still has `time_granularity`...

**Comment by szha** [comment]
Job PR-172/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-172/2/index.html

File: src/gluonts/dataset/artificial/_base.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def __init__(
         self,
         num_timeseries: int = 10,
         num_steps: int = 30,
-        time_granularity: str = "1H",
+        freq: str = "1H",
```
Comment (aalexandrov) [inline src/gluonts/dataset/artificial/_base.py]: This needs to be kept backwards compatible in the json files for now. Use

```
freq: str = pydantic.Schema(..., alias='time_granularity', schema: '1H')
```

in order to allow use with existing `metadata.json` files.

Comment (aalexandrov) [reply src/gluonts/dataset/artificial/_base.py]: FYI you can use `# type: ignore` to silence mypy for this line.

**Comment by jaheba** [comment]
Can you check the env module?

**Comment by lostella** [comment]
@jaheba I'll rebase this once #173 is merged

**Comment by aalexandrov** [comment]
> @jaheba I'll rebase this once #173 is merged

@lostella let's do it the other way around.

**Pull request merged**